eng/common/templates/post-build/channels/netcore-release-30.yml

```diff
@@ -99,7 +99,7 @@ stages:
             /p:ArtifactsCategory=$(_DotNetArtifactsCategory)
             /p:IsStableBuild=$(IsStableBuild)
             /p:IsInternalBuild=$(IsInternalBuild)
-            /p:RepositoryName='microsoft-msbuild'
+            /p:RepositoryName=$(Build.Repository.Name)
```

Comment:
@mmitche @riarenas is this safe yet? I see that dotnet/arcade#4094 is merged but is the updated package used yet? It's a workaround for dotnet/arcade#4087.

Reply:
Let me make sure that it's been updated.

Reply:
https://github.com/dotnet/arcade/pull/4145

Reply:
Thanks, we'll hold off on this update to tomorrow then.